packages/core/types/src/types/core/attributes/media.ts

```diff
@@ -25,9 +25,10 @@ export type Media<
   Attribute.WritableOption &
   Attribute.VisibleOption;
 
-// TODO: Introduce a real type for the media values
 export type MediaValue<TMultiple extends Utils.Expression.BooleanValue = Utils.Expression.False> =
-  Utils.Expression.If<TMultiple, any[], any>;
+  Attribute.GetValues<MediaTarget> extends infer TValues
+    ? Utils.Expression.If<TMultiple, TValues[], TValues>
+    : never;
 
 export type GetMediaValue<TAttribute extends Attribute.Attribute> = TAttribute extends Media<
   // Unused as long as the media value is any
```

Comment:
I wonder if we should get rid of kind altogether since it's not actually used, even now that we have a proper value

Reply:
What kind are you talking about exactly here?

Reply:
I put the comment in the wrong place sorry. I meant [this TKind](https://github.com/strapi/strapi/pull/18921/files#diff-a82648660dbae2fb4b9a91e00580cad9e9afd53e3194976be92953a2f0c2c94aR36): ['images', 'videos', 'audios', files']